metricbeat/docs/modules/linux/memory.asciidoc

```diff
@@ -9,7 +9,6 @@ beta[]
 
 include::../../../module/linux/memory/_meta/docs.asciidoc[]
 
-This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.
```

Comment:
@ruflin why is this here? It's causing the build to fail, should we remove this change?

Reply:
Interesting. I did run `make update` as the build was failing and it compalined something was missing. I guess by now this has been fixed. ++ on removing it.